src/main/java/com/amazon/ion/impl/IonReaderContinuableCoreBinary.java

```diff
@@ -1560,7 +1560,7 @@ protected PresenceBitmap loadPresenceBitmapIfNecessary(List<Macro.Parameter> sig
 
         @Override
         protected boolean isMacroInvocation() {
-            return valueTid.isMacroInvocation;
+            return valueTid != null && valueTid.isMacroInvocation;
```

Comment:
This method is now called when the cursor may be at the end of the group, at which point `valueTid` will be `null`.